### PR TITLE
CORE-12165: Update swagger baseline test, remove function from rest resource

### PR DESCRIPTION
**Pull request by anien**

### CODE REVIEW

**Review by vkolomeyko — APPROVED** [review]
LGTM

File: components/crypto/crypto-rest/src/main/java/net/corda/crypto/rest/response/package-info.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@Export
+package net.corda.crypto.rest.response;
```
Comment (dickon) [inline components/crypto/crypto-rest/src/main/java/net/corda/crypto/rest/response/package-info.java:2]: I'd normally expect to export a single package from a gradle module, so I'm surprised to see 2.

**Review by dickon — APPROVED** [review]
LGTM (the comment about exporting 2 packages seems minor).

File: components/crypto/crypto-rest/src/main/java/net/corda/crypto/rest/response/package-info.java (hunk 1 above)
Comment (anien) [reply components/crypto/crypto-rest/src/main/java/net/corda/crypto/rest/response/package-info.java:2]: This was needed for the new data class that has to be visible. I've seen two export packages per module elsewhere, hence it shouldn't be a problem.

Comment (vkolomeyko) [reply components/crypto/crypto-rest/src/main/java/net/corda/crypto/rest/response/package-info.java:2]: I would say it is perfectly normal to have more than one package exported from a module. We do have a number of other modules which do the same thing.

**Pull request merged**